search/search-dev-tools/search-dev-tools.php

```diff
@@ -100,7 +100,7 @@ function rest_callback( \WP_REST_Request $request ) {
  * @return boolean
  */
 function should_enable_search_dev_tools(): bool {
-	return ( current_user_can( SEARCH_DEV_TOOLS_CAP ) || is_debug_mode_enabled() ) && function_exists( 'ep_get_query_log' );
+	return ( current_user_can( SEARCH_DEV_TOOLS_CAP ) || is_debug_mode_enabled() );
```

Comment:
Let's rollback this change - checking for function_exists is cheap but provides extra safety (in case something will change in the future)

Reply:
Done thanks